test/locale_t.cpp

```diff
@@ -69,7 +71,7 @@ void LocaleTest::testEsperantoTranslationUtil()
 	TranslationUtil::setBaseName(test_basename);
 	
 	auto test_filename = QString { QLatin1String("some_dir/") + test_basename + QLatin1String("_eo.qm") };
-	QCOMPARE(TranslationUtil::languageFromFilename(test_filename), TranslationUtil::languageFromCode(eo));
+	QVERIFY(TranslationUtil::languageFromFilename(test_filename) == TranslationUtil::languageFromCode(eo));
```

Comment:
Unrelated change.

Reply:
Actually it is related: `QCOMPARE` was no longer able to find a comparison function. Just another case of the issues with operator lookup which consumed a lot of time when preparing this change. There might be a better fix, though.

Reply:
Solved by moving `operator==` (at the top of the file) to `namespace OpenOrienteering`.